docs/1.guide/2.event-handler.md

```diff
@@ -117,6 +117,25 @@ This will end the request with `400 - Bad Request` status code and the following
 }
 ```
 
+### String vs. Object Errors
+
+When creating an error using `createError`, you also have the option to pass a string instead of an object. Doing so will set the `message` property of the error. In this case, the `statusCode` will default to `500`.
+
+```js
+import { createError, defineEventHandler } from "h3";
+
+app.use(
+  "/hello",
+  defineEventHandler((event) => {
+    throw createError("An error occurred");
+  }),
+);
+```
+
+> [!TIP]
+> Typically, `message` contains a brief, human-readable description of the error, while `statusMessage` is specific to HTTP responses and describes the status text related to the response status code.
+> In a client-server context, using `statusMessage` is recommended because it can be accessed on the client side. Otherwise, a `message` passed to `createError` on the server will not propagate to the client.
```

Comment:
```suggestion
> In a client-server context, using a short `statusMessage` is recommended because it can be accessed on the client side. Otherwise, a `message` passed to `createError` on the server will not propagate to the client. Consider avoiding to put dynamic user input to the message to avoid potential security issues.
```